### DISCUSSION TITLE
error code 216

### DISCUSSION THREAD

**Discussion by chebbaa** [community/community#35124] [Other Feature Feedback, Questions, & Ideas]
what's the error code 216?

Can't I install it in Windows 7 version?

**Comment by radar-blip** [comment]
Looks like a virus that you dont want to install on any version of windows.

Read here: https://www.get-itsolutions.com/fix-runtime-error-216-message/

If that is what you're getting, use anti-virus software to delete it.
Good luck!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @chebbaa ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
